gunicorn.conf.py

```diff
@@ -0,0 +1,69 @@
+import uvicorn
+
+
+# Debugging
+reload = False
+reload_engine = 'auto'
+spew = False
+check_config = False
+print_config = False
+
+
+# Server socket
+bind = '127.0.0.1:8000'
+backlog = 2048
+
+
+# Worker processes
+workers = 4
+worker_class = 'uvicorn.workers.UvicornWorker'
+worker_connections = 1000
+max_requests = 0
```

Comment:
Set `max_requests` to 50-100 (reduce chance of memory leaks) in future.
Also add `max_requests_jitter` to 3-5.

Reply:
> Set `max_requests` to 50-100 (reduce chance of memory leaks) in future. Also add `max_requests_jitter` to 3-5.

Okay. I fixed all issues